### ISSUE TITLE
Course reset - Unknown column 'timeopen' in 'where clause'

### ISSUE THREAD

**Issue by cwarwicker** [markn86/moodle-mod_customcert#223]
Trying to do a course reset, with a customcert module activity:

```php
Debug info: Unknown column 'timeopen' in 'where clause'
UPDATE mdl_customcert
SET timeopen = timeopen + ?
WHERE course=? AND timeopen<>0
[array (
0 => 47605,
1 => '1311',
)]
Error code: dmlwriteexception
Stack trace:
line 489 of /lib/dml/moodle_database.php: dml_write_exception thrown
line 1139 of /lib/dml/mysqli_native_moodle_database.php: call to moodle_database->query_end()
line 5186 of /lib/moodlelib.php: call to mysqli_native_moodle_database->execute()
line 141 of /mod/customcert/lib.php: call to shift_course_mod_dates()
line 5432 of /lib/moodlelib.php: call to customcert_reset_userdata()
line 959 of /admin/tool/uploadcourse/classes/course.php: call to reset_course_userdata()
line 800 of /admin/tool/uploadcourse/classes/course.php: call to tool_uploadcourse_course->reset()
line 209 of /admin/tool/uploadcourse/classes/processor.php: call to tool_uploadcourse_course->proceed()
line 97 of /admin/tool/uploadcourse/index.php: call to tool_uploadcourse_processor->execute()
```

Moodle 3.5

**Comment by markn86** [comment]
Bah, seems there was some code I coped across that isn't relevant to this plugin. Thanks for the report!

I decided to remove the logic as these fields don't exist. Also, we shouldn't be updating the 'timecreated' or 'timemodified' values in the database for the customcert, elements, pages etc. This is only needed for plugins (like quiz) where there are 'timeopen' and 'timeclose' values that need to be shifted if the course date changes, otherwise the quiz would remained closed etc.
